### PR TITLE
Copy header Link for canonical image SEO

### PR DESCRIPTION
**Pull request by xavren**
Cf https://www.maxcdn.com/blog/better-image-seo-canonical-headers/

### CODE REVIEW

File: imageproxy.go [hunk 1]
```diff
@@ -131,7 +131,8 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	copyHeader(w, resp, "Last-Modified")
 	copyHeader(w, resp, "Expires")
 	copyHeader(w, resp, "Etag")
-
+	copyHeader(w, resp, "Link")
+
```
Comment (willnorris) [inline imageproxy.go]: remove trailing space on this line.  (or just run gofmt which will do this for you)

**Comment by xavren** [comment]
i will do a clean rebase

**Comment by willnorris** [comment]
you don't need to close this PR to do a clean rebase.  Just rebase your branch and force push to GitHub.

**Comment by xavren** [comment]
You're right, here you go

**Pull request merged**